database/postgres/pg.js

```diff
@@ -0,0 +1,51 @@
+const { Client } = require('pg')
+require('dotenv').config()
+
+const pgModule = {}
+let client
+
+pgModule.startPGDB = async ()=>{
+  try{
+    client = new Client({
+      host: process.env.HOST,
+      port: process.env.PORT,
+      user: process.env.USERNAME,
+      password: process.env.PASSWORD,
+      database: process.env.DATABASE
+    })
+    await client.connect()
+  }catch(err){
+    console.log('connection failed', err)
```

Comment:
this should return an error so whomever calls `startPGDB` will know that it has failed.

Reply:
```
return {
  error: {
    message: 'Connection to PG failed'
  }
}
```

Reply:
please add this as a test case.